plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -928,17 +928,9 @@ protected String createTableSql(RemoteTableName remoteTableName, List<String> co
     }
 
     @Override
-    protected String buildInsertSql(ConnectorSession session, RemoteTableName targetTable, RemoteTableName sourceTable, List<String> columnNames)
+    protected String postProcessInsertTableClause(ConnectorSession session)
```

Comment:
pass source sql statement as arguemnt and return postporcessed one (with suffix appended)